### PR TITLE
Map on home page

### PR DESCRIPTION
**Pull request by Suuuuha**
realted to #21

### CODE REVIEW

**Comment by Suuuuha** [comment]
should add minZoom option to change the zoom when homepage first loads

File: app/public/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./main.css">
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.3/dist/leaflet.css"
+   integrity="sha512-07I2e+7D8p6he1SIM+1twR5TIrhUQn9+I6yjqD53JQjFiMf8EtC93ty0/5vJTZGF8aAocvHYNEDJajGdNx1IsQ=="
+   crossorigin=""/>
+    <title>city fucking Nuggets</title>
```
Comment (mattlub) [inline app/public/index.html]: come on suha.

File: app/public/index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <!-- <script src='https://api.mapbox.com/mapbox-gl-js/v0.35.1/mapbox-gl.js'></script> -->
```
Comment (mattlub) [inline app/public/index.html]: remove this?

File: app/public/index.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  </head>
+  <body>
+    <div id="mapid"></div>
+
+
```
Comment (mattlub) [inline app/public/index.html]: two many empty spaces

File: app/public/main.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
+#mapid {
+  width: 500px;
+  height: 500px;
+}
+
+.map {
```
Comment (mattlub) [inline app/public/main.css]: this is not being used anywhere

File: app/public/main.css [hunk 5]
```diff
@@ -0,0 +1,13 @@
+#mapid {
+  width: 500px;
+  height: 500px;
```
Comment (mattlub) [inline app/public/main.css]: make it full screen?

File: app/public/map.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+var bounds = L.latLngBounds([32.683154, 35.278158], [32.723174, 35.341721])
+var mymap = L.map('mapid', {
+  center: [32.699, 35.303],
+  zoom: 13,
+  maxBounds: bounds
+})
+
+L.tileLayer('https://api.mapbox.com/styles/v1/mapbox/streets-v10/tiles/256/{z}/{x}/{y}?access_token=pk.eyJ1Ijoic3V1dXVoYSIsImEiOiJjajI2N3QwZm0wMDE1MnFwb3NnYnhwaG55In0.sZgdmc9B4GG9X4Bx5o3NWg', {
+  attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, <a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="http://mapbox.com">Mapbox</a>',
+  maxZoom: 18,
```
Comment (mattlub) [inline app/public/map.js]: remove max zoom from the tile layer?

File: app/public/map.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+/* global L */
+
+var bounds = L.latLngBounds([32.683154, 35.278158], [32.723174, 35.341721])
+var mymap = L.map('mapid', {
+  center: [32.699, 35.303],
+  zoom: 13,
+  maxBounds: bounds
```
Comment (mattlub) [inline app/public/map.js]: add a minzoom and maxzoom?

**Review by mattlub — CHANGES_REQUESTED** [review]

File: app/public/index.html (hunk 1 above)
Comment (Suuuuha) [reply app/public/index.html]: sorry :roll_eyes:

File: app/public/index.html [hunk 8]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1, maximum-scale=1">
+    <link rel="stylesheet" href="./main.css">
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.3/dist/leaflet.css"
+   integrity="sha512-07I2e+7D8p6he1SIM+1twR5TIrhUQn9+I6yjqD53JQjFiMf8EtC93ty0/5vJTZGF8aAocvHYNEDJajGdNx1IsQ=="
+   crossorigin=""/>
+    <title></title>
```
Comment (mattlub) [inline app/public/index.html]: add a title!!!!

File: app/public/map.js [hunk 9]
```diff
@@ -0,0 +1,15 @@
+/* global L */
+
```
Comment (mattlub) [inline app/public/map.js]: put everything inside a immediately invoked function?

e.g. var map = (function() {
...
})();

**Review by mattlub — CHANGES_REQUESTED** [review]

**Review by mattlub — APPROVED** [review]

**Pull request merged**